File: instaCult.client/src/models/Cult.js
import { Profile } from "./Account.js"


export class Cult {
  constructor(data) {
    this.name = data.name
    this.id = data.id
    this.description = data.description
    this.tags = data.tags.split(', ')
    this.leaderId = data.leaderId
    this.leader = new Profile(data.leader)
  }
}

File: instaCult/Models/Cult.cs
namespace instaCult.Models;

public class Cult : RepoItem<int>
{

  public string Name { get; set; }
  public string Description { get; set; }
  public string Tags { get; set; }
  public string LeaderId { get; set; }
  public int Popularity { get; set; }
  public Profile Leader { get; set; }

}
